aten/src/ATen/cuda/CUDAGraph.cpp

```diff
@@ -243,7 +243,7 @@ void CUDAGraph::replay() {
 }
 
 void CUDAGraph::enable_debug_mode() {
-#if !defined(USE_ROCM) || ROCM_VERSION >= 50300
+#if (!defined(USE_ROCM) || ROCM_VERSION >= 50300) || (defined(USE_ROCM) && ROCM_VERSION >= 50600)
```

Comment:
is (defined(USE_ROCM) && ROCM_VERSION >= 50600) necessary? as when if ROCM_VERSION >= 50300, this is already true.

Reply:
Don't code when tired :)  Took it off